contracts/src/Teleporter/upgrades/TeleporterUpgradeable.sol

```diff
@@ -177,6 +192,7 @@ abstract contract TeleporterUpgradeable is Context, ITeleporterReceiver, Reentra
     {
         return _pausedTeleporterAddresses[teleporterAddress];
     }
+    // solhint-enable ordering
```

Comment:
Can we re-enable ordering right after `__TeleporterUpgradeable_init`?

Reply:
nope the linter still picks up the error for the following public functions

Reply:
was wrong about this, and will be changing some of the lint disable/enables, possibly in namespace pr